### PR TITLE
Issue #14

### PR DESCRIPTION
**Pull request by kiwb**

### CODE REVIEW

File: pyclient/apps/identify_dsc.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
+import os
```
Comment (jainvipin) [inline pyclient/apps/identify_dsc.py:1]: pls put #!/usr/bin/python3 at the top of hte file and make this script an execute permissions.
Can you also include a comment about what this utility does? may be in a help string.

File: pyclient/apps/identify_dsc.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    if (len(sys.argv) == 4):
+        metricmap = {
+            "cps" : find_cps,
+            "version" : find_version,
+            "workloads" : find_workload,
+            "alerts" : find_alert
+        }
+        execfunc = metricmap.get(arg, lambda: print("Invalid Attribute"))
+        execfunc(sys.argv[2])
+    else:
```
Comment (jainvipin) [inline pyclient/apps/identify_dsc.py:26]: can we display help in this case? using argsparse would be better here. Pls see other programs that use the same.

File: pyclient/apps/identify_dsc.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+response = api_instance.list_distributed_service_card()
+telemetry_instance = TelemetryQueryV1Api(client)
+workload_instance = WorkloadV1Api(client)
+network_instance = NetworkV1Api(client)
+search_instance = SearchV1Api(client)
+
+# This function will list the DSC's whose connections per second meet the operator and value passed in by the user
+# It does this by posting a Telemtry Query for each DSC to return the CPS, and comparing this to the value
+def find_cps(oper):
+    print("List of DSC's that meet query:")
```
Comment (jainvipin) [inline pyclient/apps/identify_dsc.py:49]: minor: this comment is not useful, you can remove it.

**Review by jainvipin — CHANGES_REQUESTED** [review]
@kiwb - some comments; I am also failing here when I try to run the app
``` python3 ./apps/identify_dsc.py 
Traceback (most recent call last):
  File "./apps/identify_dsc.py", line 6, in <module>
    from utils.workload_utils import getDscFromWorkload 
ModuleNotFoundError: No module named 'utils.workload_utils'```

**Review by RishabhPatil — APPROVED** [review]
LGTM!

**Review by jainvipin — APPROVED** [review]
LGTM

**Pull request merged**